pkg/ocm/kubeletconfig_test.go

```diff
@@ -177,11 +180,47 @@ var _ = Describe("KubeletConfig", Ordered, func() {
 		Expect(err).To(HaveOccurred())
 	})
 
+	Context("List KubeletConfigs", func() {
+
+		It("Lists all kubeletconfigs", func() {
```

Comment:
We should probably have it as `KubeletConfig` just to keep it the same everywhere

Reply:
@den-rgb As per our Slack conversation, I'll fix this in the next PR which is coming shortly rather than going through the whole build for this typo 👍